### DISCUSSION TITLE
Disable version warning if run in CI

### DISCUSSION THREAD

**Discussion by AKorezin** [ansible/ansible-lint#2923] [Ideas]
It would be great if `ansible-lint` doesn't check the version if the environment variable `CI` is defined.

https://github.com/ansible/ansible-lint/blob/b7dca8c4535b2ea5d895797d8bae18e661d7726e/src/ansiblelint/app.py#L303-L306

**Comment by nobbs** [comment]
Would like to see this implemented in some way, e.g. disabling the check based on the `CI` env var or if `--offline` is used. We build images in an air-gapped environment and the `--version` call takes quite a long time to time out.
